docs/unified/novacustom/building-manual.md

```diff
@@ -0,0 +1,170 @@
+# Dasharo firmware building guide
+
+## Intro
+
+This guide shows how to build Dasharo firmware for NovaCustom devices.
+
+## Requirements
+
+This guide was verified on Ubuntu 22.04. In practice, any Linux distribution
+with [Docker](https://www.docker.com/) support should be enough to complete it.
+
+Make sure that you have following packages installed:
+
+* Docker
+    - follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    - follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+* Git
+
+    ```bash
+    sudo apt -y install git
+    ```
+
+## Build Dasharo BIOS firmware
+
+1. Refer to the table below to set the `SDK`, `REVISION`, and `CONFIG` variables
+   based on your device and preference:
+    - using release branch will always result in the latest released version,
+    - using tag you can specify the exact version to build.
+
+| Device        | SDK version             | release branch | tag format                 | config |
+|---------------|-------------------------|----------------|----------------------------|---------------------|
+| NV4X TGL      | coreboot-sdk:0ad5fbd48d | clevo/release  | novacustom_nv4x_vX.Y.Z     | novacustom_nv4x_tgl |
+| NS5X/7X TGL   | coreboot-sdk:0ad5fbd48d | clevo/release  | novacustom_ns5x_vX.Y.Z     | novacustom_ns5x_tgl |
+| NS5X/NS7X ADL | coreboot-sdk:0ad5fbd48d | clevo/release  | novacustom_ns5x_adl_vX.Y.Z | novacustom_nv4x_adl |
+| NV4X ADL      | coreboot-sdk:0ad5fbd48d | clevo/release  | novacustom_nv4x_adl_vX.Y.Z | novacustom_ns4x_adl |
```

Comment:
- coreboot-sdk is not a variable here, I don't see need for that information it is redundant
- same thing with release branch

Reply:
yes I wanted to extend it to more platforms than NovaCustom but stopped there to review it to see if that is a good path forward (i'm thinking maybe not the best one)

Reply:
if we want to do such experiments we should gather all combinations in one spreadsheet, it would be good exercise and will definitely show some patterns and some platforms which may need improvements and unification, question if we want to perform unification now or in future - answer is budget

also, cost of not compliant releases may be higher then those keeping unification

Reply:
Table no longer exists, this can be resolved I think